patches/chrome-android-java-res-values-v17-styles.xml.patch

```diff
@@ -1,10 +1,11 @@
 diff --git a/chrome/android/java/res/values-v17/styles.xml b/chrome/android/java/res/values-v17/styles.xml
-index 5b41c992fb2b17c1b8a8a06ecc09460c4190fa86..b4a36c8d48dcc89a1dd0e9c661f4d3eedc757ed8 100644
+index 5b41c992fb2b17c1b8a8a06ecc09460c4190fa86..31ba6cbbfd63ece1bb618483d286438d7e8fc8cc 100644
 --- a/chrome/android/java/res/values-v17/styles.xml
 +++ b/chrome/android/java/res/values-v17/styles.xml
-@@ -995,4 +995,5 @@
+@@ -995,4 +995,6 @@
          <item name="android:textAlignment">viewStart</item>
          <item name="android:textAppearance">@style/TextAppearance.BlackTitle1</item>
      </style>
 +    <style name="BraveSwitchTheme"> <item name="colorControlActivated">#fb542b</item> </style>
++    <style name="BraveDialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert"> <item name="windowNoTitle">true</item> </style>
```

Comment:
Just put that string to the end of previous string to make the patch one line. We don't care that it looks ugly, we want to have as less lines as we can in patches.